haystack/components/embedders/sentence_transformers_text_embedder.py

```diff
@@ -47,26 +48,27 @@ def __init__(
         Create a SentenceTransformersTextEmbedder component.
 
         :param model:
-            Local path or ID of the model on HuggingFace Hub.
+            The model to use for calculating embeddings. 
+            Specify the path to a local model or the ID of the model on Hugging Face.
         :param device:
             Overrides the default device used to load the model.
         :param token:
-            The API token used to download private models from Hugging Face.
+            To use private models from Hugging Face, enter an API token here.
         :param prefix:
-            A string to add at the beginning of each text.
-            Can be used to prepend the text with an instruction, as required by some embedding models,
+            A string to add at the beginning of each text to be embedded.
+            You can use it to prepend the text with an instruction, as required by some embedding models,
             such as E5 and bge.
         :param suffix:
-            A string to add at the end of each text.
+            A string to add at the end of each text to embed.
         :param batch_size:
-            Number of Documents to encode at once.
+            Number of texts to embed at once.
         :param progress_bar:
-            If True shows a progress bar when running.
+            Enables a progress bar for calculating embeddings.
```

Comment:
How about we use the format as in lines 70-71? "If True, enables a progress bar"
Then users won't need to dig up the parameter type

Reply:
makes sense!